### PR TITLE
bug fix to allow batch/template with one field

### PR DESCRIPTION
**Pull request by bethbrains**
# Critical Changes

# Changes
- Fixed bug to allow single-field batch/template in BGE

# Issues Closed

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by bethbrains** [comment]
**lurch: attach W-029642

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000007fPO9QAM" target="_blank">W-029642</a>

File: src/aura/BGE_EntryForm/BGE_EntryFormHelper.js [hunk 1]
```diff
@@ -87,13 +87,17 @@
      */
     verifyRequiredFields: function(component, rowFields) {
         var missingFields = [];
-        var dataImportFields = component.get("v.dataImportFields");
-        var dynamicInputFields = component.find("dynamicInputFields");
+        var dataImportFields = component.get('v.dataImportFields');
+        var dynamicInputFields = component.find('dynamicInputFields');
+
+        if (! Array.isArray(dynamicInputFields)) {
```
Comment (lmeerkatz) [inline src/aura/BGE_EntryForm/BGE_EntryFormHelper.js]: I'd take out the extra space after !, but other than that it's perfect. :)

**Comment by CarmeNima** [comment]
QA has Passed.

**Pull request merged**